### ISSUE TITLE
Share button is not accessible to screen reader

### ISSUE THREAD

**Issue by cassidyjames** [ubuntu/app-center#1589]
When I focus an app detail page's share button with the keyboard when using Ubuntu's built-in screen reader, it just reads, "push button" instead of "share button," "copy URL," or similar.
